docs/source/clusters.rst

```diff
@@ -23,90 +25,153 @@ Which one you choose is largely dependent on your processes, requirements and ho
-            "current_version": "$LATEST",
-            # Required if custom SNS inputs as configured
-            "inputs": {
-              "aws-sns:": []
-            }
-          }
-        },
-        "cloudwatch_monitoring": {
-          "enabled": true
+  {
+    "id": "production",
+    "region": "us-west-2",
+    "modules": {
+      "stream_alert": {
+        "alert_processor": {
+          "timeout": 25,
+          "memory": 128,
+          "current_version": "$LATEST"
         },
-        "kinesis": {
-          "streams": {
-            "shards": 1,
-            "retention": 24
-          },
-          "firehose": {
-            "enabled": true,
-            "s3_bucket_suffix": "streamalert.results"
-          }
-        },
-        "kinesis_events": {
-          "enabled": true
-        },
-        # Optional - Configure CloudTrail into Kinesis
-        "cloudtrail": {
-          "enabled": true,
-          "event_pattern": {
-            "source": [],
-            "detail-type": [],
-            "detail": {}
-          }
+        "rule_processor": {
+          "timeout": 10,
+          "memory": 256,
+          "current_version": "$LATEST"
+        }
+      },
+      "cloudwatch_monitoring": {
+        "enabled": true
+      },
+      "kinesis": {
+        "streams": {
+          "shards": 1,
+          "retention": 24
         },
-        # Optional - Configure VPC Flow Logs into Kinesis
-        "flow_logs": {
+        "firehose": {
           "enabled": true,
-          "vpcs": [],
-          "subnets": [],
-          "enis": []
+          "s3_bucket_suffix": "streamalert.results"
         }
       },
-      # Optional Terraform Outputs
-      "outputs": {
-        "kinesis": [
-          "username",
-          "access_key_id",
-          "secret_key"
-        ]
+      "kinesis_events": {
+        "enabled": true
       }
+    },
+    "outputs": {
+      "kinesis": [
+        "username",
+        "access_key_id",
+        "secret_key"
+      ]
     }
+  }
+
+Customizing Clusters
+~~~~~~~~~~~~~~~~~~~~
+
+Each cluster can be broken up into multiple 'odules to make up a StreamAlert cluster.
+
+Each module corresponds to a Terraform module found in the ``terraform/modules`` directory, and serves a specific purpose in a StreamAlert cluster.
+
+After making modifications to a cluster's file, make sure you apply it with::
+
+  $ python stream_alert_cli.py terraform build
+  
+This will regenerate the necessary Terraform files and then apply the changes.
+
+Module: StreamAlert
+--------------------
+
+See `Lambda Settings <lambda.html>`_ for customization options.
+
+Module: Kinesis
+---------------
+
+See `Kinesis <kinesis.html>`_ for customization options.
+
+Module: CloudWatch Monitoring
+-----------------------------
+
+Amazon CloudWatch is a monitoring service for AWS cloud resources.
+
+To ensure a StreamAlert cluster is running properly, this module creates metric based alarms across various running services.
```

Comment:
"metric based alarms across various running services."

only services that streamalert stood up or? (add this clarity and speak to the benefits briefly)